lua/lv-which-key/init.lua

```diff
@@ -23,8 +23,8 @@ which_key.setup {
     },
   },
   icons = {
-    breadcrumb = "»", -- symbol used in the command line area that shows your active key combo
-    separator = "➜", -- symbol used between a key and it's label
+    breadcrumb = "¬ª", -- symbol used in the command line area that shows your active key combo
+    separator = "‚ûú", -- symbol used between a key and it's label
```

Comment:
can you revert these two lines, seems like unrelated to this PR

Reply:
Nice catch. Done.